transform/operations.go

```diff
@@ -239,11 +239,32 @@ func (t *timeParse) transform(raw interface{}) (interface{}, error) {
 	if !ok {
 		return nil, errors.New("timeParse only supports strings")
 	}
-	parsedTime, err := time.Parse(t.Args["format"], in)
+	parsedTime, err := time.Parse(t.args["format"], in)
 	if err != nil {
 		return nil, fmt.Errorf("time could not be parsed using supplied format")
 	}
-	return parsedTime.Format(t.Args["layout"]), nil
+	return parsedTime.Format(t.args["layout"]), nil
+}
+
+type currentTime struct {
+	args map[string]string
+}
+
+func (c *currentTime) init(args map[string]string) error {
+	if err := requiredArgs([]string{"format"}, args); err != nil {
+		return err
+	}
+	c.args = args
+	return nil
+}
+
+func (c *currentTime) transform(raw interface{}) (interface{}, error) {
+	timeFmt := c.args["format"]
+	switch c.args["format"] {
```

Comment:
Would you want a `default` case to this switch statement?

Reply:
hypothetically the default is rfc3339. Anything else is up to the peoples.